src/features/embalm/stepContent/steps/SelectArchaeologists.tsx

```diff
@@ -1,22 +1,226 @@
+  HStack,
+  Icon,
+  Box,
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverBody,
+} from '@chakra-ui/react';
+import React, { ChangeEvent } from 'react';
+import { SummaryErrorIcon } from '../components/SummaryErrorIcon';
 import { ArchaeologistList } from '../components/ArchaeologistList';
+import { ArchaeologistHeader } from '../components/ArchaeologistHeader';
+import {
+  Pagination,
+  usePagination,
+  PaginationNext,
+  PaginationPage,
+  PaginationSeparator,
+  PaginationPrevious,
+  PaginationContainer,
+  PaginationPageGroup,
+} from '@ajna/pagination';
+import { useArchaeologistList } from '../hooks/useArchaeologistList';
+import { ChevronLeftIcon, ChevronRightIcon, QuestionIcon } from '@chakra-ui/icons';
 
 export function SelectArchaeologists() {
-  const { selectedArchaeologists } = useSelector(x => x.embalmState);
+  const outerLimit = 1;
+  const innerLimit = 1;
+  const { sortedFilteredArchaeologist } = useArchaeologistList();
+
+  const { currentPage, setCurrentPage, pagesCount, pages, pageSize, setPageSize, offset } =
+    usePagination({
+      total: sortedFilteredArchaeologist.length,
+      initialState: { currentPage: 1, pageSize: 5 },
+      limits: {
+        outer: outerLimit,
+        inner: innerLimit,
+      },
+    });
+
+  const paginatedArchaeologist = sortedFilteredArchaeologist.slice(offset, offset + pageSize);
+
+  const handlePageSizeChange = (event: ChangeEvent<HTMLSelectElement>): void => {
+    const newPageSize = Number(event.target.value);
+    setPageSize(newPageSize);
+  };
+
+  const handlePageChange = (nextPage: number): void => {
+    setCurrentPage(nextPage);
+  };
+
+  const returnToFirstPage = () => {
+    setCurrentPage(1);
+  };
 
   return (
     <Flex
       direction="column"
       width="100%"
     >
       <Heading>Select Archaeologists</Heading>
-      <Text variant="secondary">Selected archaeologists: {selectedArchaeologists.length}</Text>
-      <Text variant="secondary">
-        Total digging fee {sumDiggingFees(selectedArchaeologists).toString()} SARCO.
+      <Text
+        variant="primary"
+        mt="6"
+      >
+        Resurrection Time
       </Text>
-      <ArchaeologistList />
+      <Text
+        variant="primary"
+        mt="2"
+      >
+        Currently set: 09.22.22 7:30pm (edit)
+      </Text>
+      <ArchaeologistHeader resetPage={returnToFirstPage} />
+      <Pagination
+        pagesCount={pagesCount}
+        currentPage={currentPage}
+        onPageChange={handlePageChange}
+      >
+        <PaginationContainer
+          p={4}
+          w="full"
+        >
+          <VStack>
+            <ArchaeologistList paginatedArchaeologist={paginatedArchaeologist} />
+            <Box w={'100%'}>
+              <Flex justifyContent={'space-between'}>
+                <Flex px={3}>
+                  <HStack direction="row">
+                    <HStack>
+                      <Text color="brand.600">Items per page:</Text>
+                      <Select
```

Comment:
This `Select` needs to be styled. These `Select` components are a pain to style properly, which is why we have a library for it `chakra-react-select`, but it's not straightforward. This work should be done in a different task.

![image](https://user-images.githubusercontent.com/34484576/201952881-fb303dd1-822f-4ba4-a135-62f6e7e76a75.png)

Reply:
Yes, I was planning to address it in the next PR I'm currently working on